ui-charts/src/spaceTimeChart/components/TimeCaptions.tsx

```diff
@@ -51,14 +61,19 @@ const TimeCaptions = () => {
           timeCaptionsPriorities,
           timeCaptionsStyles,
           timeGraduationsStyles,
+          dateCaptionsStyle,
         },
+        captionSize,
+        hideDates,
         showTicks,
       }
     ) => {
       const timeAxisSize = !swapAxis ? width : height;
       const spaceAxisSize = !swapAxis ? height : width;
```

Comment:
Why not subtract captionSize here ?
```suggestion
      const spaceAxisSize = (!swapAxis ? height : width) - captionSize;
```

Reply:
Good catch, I just pushed a fix for this. Thanks!